webview/src/experiments/components/table/Table.test.tsx

```diff
@@ -434,4 +422,36 @@ describe('Table', () => {
       )
     })
   })
+  describe('Focused Column', () => {
```

Comment:
First time writing tests! Do they look ok 😅

Reply:
These are great. And thanks for reminding me about `toHaveClass`. I have not used it in the project, I used a messier way (with some `eslint-disable`).